types/api.go

```diff
@@ -195,3 +211,124 @@ type Key struct {
 	Idx int    `json:"idx"`
 	Key string `json:"key"`
 }
+
+// VochainStats contains information about the current Vochain statistics and state
+type VochainStats struct {
+	BlockHeight      uint32    `json:"block_height"`
+	EntityCount      int64     `json:"entity_count"`
+	EnvelopeCount    uint64    `json:"envelope_count"`
+	ProcessCount     int64     `json:"process_count"`
+	ValidatorCount   int       `json:"validator_count"`
+	BlockTime        *[5]int32 `json:"block_time"`
+	BlockTimeStamp   int32     `json:"block_time_stamp"`
+	ChainID          string    `json:"chain_id"`
+	GenesisTimeStamp time.Time `json:"genesis_time_stamp"`
+
+	// TransactionHeight int64     `json:"transaction_height"`
+	// AvgTxsPerBlock    float64   `json:"avg_txs_per_block"`
+	// AvgTxsPerMinute   float64   `json:"avg_txs_per_minute"`
+	// The hash of the block with the most txs
+	// MaxBytes          int64  `json:"max_bytes"`
+	// MaxTxsBlockHash   string `json:"max_txs_block_hash"`
+	// MaxTxsBlockHeight int64  `json:"max_txs_block_height"`
+	// The start of the minute with the most txs
+	// MaxTxsMinute    time.Time `json:"max_txs_minute"`
+	// MaxTxsPerBlock  int64     `json:"max_txs_per_block"`
+	// MaxTxsPerMinute int64     `json:"max_txs_per_minute"`
```

Comment:
what's the point of this chunk of commented out code? if you really want to keep it around, maybe add a TODO explaining why